AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -648,6 +648,7 @@ reddit.com#@#.ADBAR
 ! SECTION: Anti-adblock - Regular rules
 ! Do not forget about '!#safari_cb_affinity' section for exclusions
 !
+cizgiromanarsivi.com##.popup53984wrap
```

Comment:
Maybe they will change the number from time to time, let's try:
`cizgiromanarsivi.com##div[id][class^="popup"][class$="wrap"]`